docs/enable-extensions.md

```diff
@@ -10,7 +10,7 @@ While setting up a high availability PostgreSQL cluster with Patroni, you will n
 
 - Patroni installed on every ``postresql`` node. 
 
-- Distributed Configuration Store (DCS). Patroni supports such DCSs as ETCD, zookeeper, Kubernetes though [ETCD](https://etcd.io/) is the most popular one. It is available upstream as DEB packages for Debian 10, 11 and Ubuntu 18.04, 20.04, 22.04.  
+- Distributed Configuration Store (DCS). Patroni supports such DCSs as ETCD, zookeeper, Kubernetes though [ETCD](https://etcd.io/) is the most popular one. It is available upstream as DEB packages for Debian 10, 11, 12 and Ubuntu 18.04, 20.04, 22.04.  
```

Comment:
@Naeem-Akhter @EvgeniyPatlan please comment

Reply:
Yes, Ubuntu 18.04 is EOL and, where needed, we need to mention that we are not shipping packages for this platform any more since May, 2023. Please also exclude it from deb packages list given above.